### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by dpshekhawat**
PR for updating the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
 
 **Why for Edge Devices?** -
-The need for on-device data analysis arises in cases where decisions based on data processing have to be made immediately. For example, there may not be sufficient time for data to be transferred to back-end servers, or there is no connectivity at all.
-However, with the arrival of powerful, low-energy consumption Internet of Things devices, computations can now be executed on edge devices such as robots themselves. This has given rise to the era of deploying advanced machine learning methods at the edges of the network for “edge-based” ML.
+The need for on-device data analysis arises in cases where decisions based on data processing have to be made immediately. With the arrival of powerful, low-energy consumption Internet of Things devices, computations can now be executed on edge devices such as robots themselves. This has given rise to the era of deploying advanced machine learning methods at the edges of the network for “edge-based” ML.
```
Comment (pkliczewski) [inline README.md]: I would remove `themselves` and expand `ML` to machine learning.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
 
 ### Model Used 
-State-of-the-art lightweight Yolov4-tiny model is especially useful if you have limited compute resources in either research or deployment, and are willing to tradeoff some detection performance for speed. It will display the predicted classes as well as the image with bounding boxes drawn on top of it.
+State-of-the-art lightweight Yolov4-tiny model is especially useful if you have limited compute resources in either research or deployment, and are willing to tradeoff some detection performance for speed.
```
Comment (pkliczewski) [inline README.md]: What do you think about linking additional information about the model?

File: README.md [hunk 3]
```diff
@@ -1,32 +1,57 @@
...
   - Python >=3.7
   - OpenCV 4.6
   
->Docker and Kubernetses required.
+>Docker and Kubernetes required.
 
 ## Run
-  
-You will need a webcam mounted to your workload or it fails with no device connected. Change the device id from 0 to 1/2/3... specific to your requirements
+***Note: For detailed description visit [Flotta](https://project-flotta.io/blog.html)***
```
Comment (pkliczewski) [inline README.md]: What do you think about using this [address](https://project-flotta.io/documentation/latest/intro/overview.html) instead?

File: README.md [hunk 4]
```diff
@@ -59,7 +79,45 @@ bag
...
+      spec:
+        containers:
+           - name: edge-ic-workload
+             image: quay.io/dpshekhawat/img-class:latest #change here
+    ```
+2. Using as is:
+    
+    Clone the github repository and start deploying the workload on the devices.
+
+Learn more on running workloads [here](https://project-flotta.io/documentation/v0_1_0/gsg/running_workloads.html).
```
Comment (pkliczewski) [inline README.md]: I would link to the [latest](https://project-flotta.io/documentation/latest/gsg/running_workloads.html) documentation

File: README.md (hunk 3 above)
Comment (dpshekhawat) [reply README.md]: I was thinking about adding the blogpost of image classification when published referenced here

Comment (pkliczewski) [reply README.md]: sounds like good idea

**Comment by dpshekhawat** [comment]
@pkliczewski please have a final look

**Review by pkliczewski — APPROVED** [review]
Looks good to me! @dpshekhawat should I merge?

**Comment by dpshekhawat** [comment]
Sure, it's ready to merge!

**Pull request merged**